WebRoot/templates/calendar/Appointment.template

```diff
@@ -995,16 +996,94 @@
 			</tr>
 		<$
 			} else {
-				var ptstStatus = data.ptstStatus;
-				for (var i in ptstStatus) {
-					if (!ptstStatus[i].attendees) { continue; }
+				var attendeesPtStStatus = data.attendeesPtStStatus;
+				for (var i in attendeesPtStStatus) {
+					if (!attendeesPtStStatus[i].attendees) { continue; }
 		$>
 				<tr valign='top'>
 					<td align='right' style='padding-right:5px;' nowrap>
-						<b><$= AjxMessageFormat.format(ZmMsg.makeLabel, AjxStringUtil.htmlEncode(i) + "&nbsp;(&nbsp;" + ptstStatus[i].count + "&nbsp;)&nbsp;") $></b>
+						<b><$= AjxMessageFormat.format(ZmMsg.makeLabel, AjxStringUtil.htmlEncode(i) + "&nbsp;(&nbsp;" + attendeesPtStStatus[i].count + "&nbsp;)") $></b>
 					</td>
 					<td nowrap>
-						<$= AjxStringUtil.htmlEncode(ptstStatus[i].attendees) $>
+						<$= AjxStringUtil.htmlEncode(attendeesPtStStatus[i].attendees) $>
+					</td>
+				</tr>
+		<$
+				}
+			}
+	}
+	$>
+
+	<$ if (!data.appt.isDraft && data.appt.isOrganizer() && data.appt.otherAttendees && !data.hideLocations) { $>
```

Comment:
we are duplicating same block of code, please check to reduce duplication

Reply:
reduced code